lib/vagrant-linode/actions/create.rb

```diff
@@ -29,6 +29,23 @@ def call(env)
             root_pass = Digest::SHA2.new.update(@machine.provider_config.api_key).to_s
           end
 
+          if @machine.provider_config.stackscript
+            stackscripts = @client.stackscript.list + @client.avail.stackscripts
+            stackscript = stackscripts.find { |s| s.label.downcase == @machine.provider_config.stackscript.to_s.downcase }
```

Comment:
I'd like to see us find from the private list, then find from the public list if there was no private match.